deps/rabbitmq_event_exchange/src/rabbit_exchange_type_event.erl

```diff
@@ -107,10 +107,7 @@ code_change(_OldVsn, State, _Extra) -> {ok, State}.
 %%----------------------------------------------------------------------------
 
 ensure_vhost_exists(VHost) ->
-    case rabbit_vhost:exists(VHost) of
-        false -> rabbit_vhost:add(VHost, ?INTERNAL_USER);
-        _     -> ok
-    end.
+    rabbit_vhost:add(VHost, ?INTERNAL_USER).
```

Comment:
Have you measured this?

Reply:
The concern here is that it (counterintuitively) may make definition import slower